Tasks/ExtractFiles/extractfilestask.ts

```diff
@@ -88,7 +90,8 @@ function findFiles(): string[] {
             // ensure only files are added, since our search results may include directories
             for (var j = 0; j < matched.length; j++) {
                 var match = path.normalize(matched[j]);
-                if (fs.lstatSync(match).isFile()) {
+                var stats = tl.stats(match);
+                if (stats.isFile()) {
```

Comment:
We should be OK here because the underlying implementation passes through to fs.stats() -- which has two possibilities, one that returns a valid stats object, or a callback for exceptions -- in the case of a bad match, this will throw an exception out and fail that task with an error message like this (I just tried this to make sure): 
about to match foo
Unhandled: ENOENT: no such file or directory, stat 'C:\agents\latest\tasks\ExtractFiles\1.0.1\foo'